bibtexparser/bwriter.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, write_common_strings=False):
         #: Align values. Determines the maximal number of characters used in any fieldname and aligns all values
         #    according to that by filling up with single spaces. Default: False
         self.align_values = False
+        # Align multi-line values. Formats a multi-line value such that the text is aligned exactly on top of each other.
```

Comment:
```suggestion
        #: Align multi-line values. Formats a multi-line value such that the text is aligned exactly
        #    on top of each other. Default: False
```

Just to be consistent with the remainder of the docstring